src/MoviePage/MovieItem/index.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import {
+  itemCard,
+  movieImage,
+  movieInfo,
+  movieTitle,
+  movieOverview
+} from "./style";
+
+/**
+ * Render the data of a movie
+ *
+ * @param {array} movie - Data from a movie
+ * @returns {object}
+ */
+function MovieItem({ movie = [] }) {
```

Comment:
An issue with `eslint` here

Reply:
The `prop-types` rule that I ignore on the `eslint` configuration was still on but is already fixed

Reply:
Fix the issue. Was a typo on the rule 😅